resolwe/flow/models/entity.py

```diff
@@ -26,3 +27,39 @@ class Meta(BaseCollection.Meta):
 
     #: indicate whether `descriptor` is completed (set by user)
     descriptor_completed = models.BooleanField(default=False)
+
+
+class RelationType(models.Model):
+    name = models.CharField(max_length=100, unique=True)
+
+    ordered = models.BooleanField(default=False)
```

Comment:
If it is ordered, what is the order key? Is it "position"? How is it ordered (e.g. what is the comparator)?